components/Checkbox/src/index.svelte

```diff
@@ -27,14 +26,17 @@
 </script>
 
 <label
-	class="k-checkbox--base k-checkbox--base__dark {cnames} {disabled ? 'k-cur-disabled' : ''}"
+	class={clsx(cls, 'k-checkbox--base k-checkbox--base__dark', {
```

Comment:
https://github.com/ikun-svelte/ikun-ui/blob/main/components/Button/src/index.svelte#L66

I think you should refer to the button component writing method instead of using `clsx` directly in html.I think we should gradually standardize and unify this writing style.
wdyt？

Reply:
Totally agree with you, but the file has got three classes and each one needs to have a separate variable, and each classname isn't that long, creating variable for each class is a bad idea, instead use them as-is and this pattern is very well-known in react.

QA: Will the function get re-called each time it renders ? No, basically svelte handles such cases so it will be exactly the same as `$: cnames = clsx(...)`.

Looking to hear from you.